bucket/pypy3.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "7.1.1",
+    "license": "MIT",
+    "extract_dir": "pypy3.6-v7.1.1-win32",
+    "url": "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.1.1-win32.zip",
+    "homepage": "https://pypy.org",
+    "hash": "8b513b254de5f31890f5956569de9aec3a0a91d7aba72fc89d66901f4a8ccf49",
+    "bin": [
+        [
+            "pypy3.exe",
+            "pypy3"
+        ],
+        [
+            "pypy3w.exe",
+            "pypy3w"
+        ]
+    ],
+    "checkver": {
+        "url": "https://bitbucket.org/pypy/pypy/downloads/",
+        "re": "pypy3.6-v([\\d.]+)-win32"
+    },
+    "autoupdate": {
+        "url": "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v$version-win32.zip",
+        "extract_dir": "pypy3.6-v$version-win32"
+    }
```

Add hash extraction from https://pypy.org/download.html

Should be enough
```suggestion
        "hash": {
            "url": "https://pypy.org/download.html"
        }
    }
```